app/index.js

```diff
@@ -110,6 +110,19 @@ app.model = require('./model');
 app.controller = require('./controller');
 
 /**
+ * Log potentially unhandled promises
+ */
+// TODO: move over to testing suite
+if (app.config.env === 'development') {
+  process.on('unhandledRejection', function(reason, p) {
```

Comment:
It is a catch-all. I'm not sure what you mean by an express middleware for the catch-all.